### PR TITLE
Chore/tracefoss 1968 configure int to use official discoveryfinder

### PR DESCRIPTION
**Pull request by ds-ext-sceronik**

### CODE REVIEW

File: charts/connector/edc-provider/values-int-a.yaml [hunk 1]
```diff
@@ -66,16 +76,13 @@ edc-postgresql:
   primary:
     resources:
       limits:
-        cpu: 2000m
-        memory: 4Gi
-      requests:
-        cpu: 100m
-        memory: 4Gi
+        cpu: 1500m
```
Comment (ds-mwesener) [inline charts/connector/edc-provider/values-int-a.yaml:79]: That is a lot request. Can we not go with 100m and set the limits to 1500m if necessary? Please make sure to always use same value on ram.

**Review by ds-mwesener — CHANGES_REQUESTED** [review]
Please address comments. Thanks!

File: charts/connector/edc-provider/values-int-a.yaml (hunk 1 above)
Comment (ds-ext-sceronik) [reply charts/connector/edc-provider/values-int-a.yaml:79]: Indeed was testing provider and if the exception issue is resolved after bumping up resource usage . did not help so I will roll that back

File: charts/connector/edc-provider/values.yaml [hunk 2]
```diff
@@ -164,10 +164,10 @@ tractusx-connector:
 edc-postgresql:
   primary:
     resources:
-      limits:
-        cpu: 2000m
-        memory: 4Gi
       requests:
         cpu: 100m
-        memory: 4Gi
+        memory: 1.5Gi
```
Comment (ds-mwesener) [inline charts/connector/edc-provider/values.yaml:169]: Ram needs to be same value as limit

**Review by ds-mwesener — APPROVED** [review]
LGTM

**Pull request merged**